src/Service.Host/package.json

```diff
@@ -65,12 +65,12 @@
         "jest-environment-jsdom": "^29.0.3",
         "jest-environment-jsdom-sixteen": "^2.0.0",
         "jest-transform-stub": "^2.0.0",
-        "node-sass": "^6.0.1",
         "postcss": "^8.3.11",
         "postcss-loader": "^6.2.0",
         "prettier": "^2.4.1",
         "raf": "^3.4.1",
         "regenerator-runtime": "^0.13.9",
+        "sass": "^1.66.1",
```

Comment:
npm install wouldn't work on my mac without this package.json change... Hope it doesn't break it everywhere else!